src/routes/docs/products/auth/team-invites/+page.markdoc

```diff
@@ -0,0 +1,715 @@
+    .setEndpoint("https://cloud.appwrite.io/v1")
+    .setProject("<PROJECT_ID>")
+
+val teams = Teams(client)
+
+// Accept the invitation using the membership ID and secret
+val response = teams.updateMembershipStatus(
+    teamId = "team_id",
+    membershipId = "membership_id",
+    userId = "user_id",
+    secret = "secret"
+)
+```
+{% /multicode %}
+
+# Server-side custom flows {% #server-side %}
+
+Server-side membership creation bypasses the email invitation process, allowing direct member addition. This approach:
+1. Creates an active membership immediately
+2. Doesn't require user acceptance
+3. Gives you complete control over the invitation workflow
+
+This makes them perfect for scenarios requiring custom workflows, such as bulk user management, automated team assignments, or integration with external systems.
+Since memberships are created directly, users gain immediate access without waiting for email acceptance.
+
+{% multicode %}
+```server-nodejs
+const sdk = require('node-appwrite');
+
+const client = new sdk.Client()
+    .setEndpoint('https://cloud.appwrite.io/v1')
+    .setProject('<PROJECT_ID>')
+    .setKey('<API_KEY>');
+
+const teams = new sdk.Teams(client);
+
+// Create membership directly with userId
+const membership = await teams.createMembership(
+    'team_id',
+    ['developer'],
+    'user_id',
+    'John Doe'  // optional name
+);
+```
+```server-python
+from appwrite.client import Client
+from appwrite.services.teams import Teams
+
+client = Client()
+client.set_endpoint('https://cloud.appwrite.io/v1')
+client.set_project('<PROJECT_ID>')
+client.set_key('<API_KEY>')
+
+teams = Teams(client)
+
+# Create membership directly with userId
+membership = teams.create_membership(
+    team_id='team_id',
+    roles=['developer'],
+    user_id='user_id',
+    name='John Doe'  # optional
+)
+```
+```server-swift
+import Appwrite
+
+let client = Client()
+    .setEndpoint("https://cloud.appwrite.io/v1") // Your API Endpoint
+    .setProject("<PROJECT_ID>") // Your project ID
+    .setKey("<API_KEY>") // Your secret API key
+
+let teams = Teams(client)
+
+// Create membership directly with userId
+let membership = try await teams.createMembership(
+    teamId: "team_id",
+    roles: ["developer"],
+    userId: "user_id",
+    name: "John Doe" // optional
+)
+```
+```server-kotlin
+import io.appwrite.Client
+import io.appwrite.services.Teams
+
+val client = Client()
+    .setEndpoint("https://cloud.appwrite.io/v1") // Your API Endpoint
+    .setProject("<PROJECT_ID>") // Your project ID
+    .setKey("<API_KEY>") // Your secret API key
+
+val teams = Teams(client)
+
+// Create membership directly with userId
+val response = teams.createMembership(
+    teamId = "team_id",
+    roles = listOf("developer"),
+    userId = "user_id",
+    name = "John Doe" // optional
+)
+```
+{% /multicode %}
+
+# Manage memberships {% #manage-memberships %}
+
+## Check membership status {% #status %}
+
+### Client-side
```

Comment:
Does this look odd to you?

<img width="719" alt="image" src="https://github.com/user-attachments/assets/d0384e21-c502-4611-a420-d6e86cc1e851" />

Reply:
Updated